test/infrastructure/docker/internal/loadbalancer/config.go

```diff
@@ -18,16 +18,19 @@ package loadbalancer
 
 import (
 	"bytes"
+	"net"
 	"text/template"
 
 	"sigs.k8s.io/kind/pkg/errors"
 )
 
 // ConfigData is supplied to the loadbalancer config template.
 type ConfigData struct {
-	ControlPlanePort int
-	BackendServers   map[string]string
-	IPv6             bool
+	FrontendControlPlanePort string
+	BackendControlPlanePort  string
+	BackendServers           map[string]string
+	IPv6                     bool
+	UnsafeLoadBalancerConfig string
```

Comment:
This feels like it shouldn't live on this struct, as this struct is use when templating and contains data to be used within a template.